staging/kos/pkg/controllers/subnet/validator.go

```diff
@@ -478,3 +645,11 @@ func equal(s1, s2 []string) bool {
 	}
 	return true
 }
+
+// TODO move to shared util pkg
+func FormatErrVal(err bool) string {
```

Comment:
Maybe include the abstraction function (`!= nil`) too.

Reply:
See how at https://github.com/MikeSpreitzer/kube-examples/blob/make-subnet-validator-observable/staging/kos/pkg/controllers/subnet/validator.go#L575 that function is used with an argument other than the expression `err != nil`. We can change it to accept an interface argument and check for nilness inside, but then clients which pass a slice in must make sure that it's not empty. I have no preference between keeping things as they are or doing this change.

Reply:
Oh, right.  I think it's OK as it is.